src/main/java/com/sigmaukraine/messenger/repository/ChatRepository.java

```diff
@@ -0,0 +1,123 @@
+package com.sigmaukraine.messenger.repository;
+
+import com.sigmaukraine.messenger.domain.Chat;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Repository
+@Transactional
+public class ChatRepository {
+    private static Logger LOG = Logger.getLogger(ChatRepository.class.getName());
+
+    @Autowired
+    private SessionFactory sessionFactory;
+
+    /**
+     * Returns list of all chats
+     * @return List<Chat>
+     */
+    public List<Chat> listAll() {
+        if (LOG.isLoggable(Level.INFO)) {
```

Comment:
Consider switching to logback because java.util.logging is awful. You don't need such clutter checks with logback